jenkins/jobs/integration_tests.pipeline

```diff
@@ -38,6 +38,8 @@ pipeline {
     OS_IDENTITY_API_VERSION=3
     TEST_EXECUTER_VM_NAME = "${VM_NAME}"
     BUILD_URL = "${env.BUILD_URL}"
+    BUILD_NUMBER = "${env.BUILD_NUMBER}"
+    JOB_NAME = "${env.JOB_BASE_NAME}"
```

Comment:
you could probably use BUILD_TAG which covers both BUILD_NUMBER and JOB_NAME.

Reply:
done, thank you